src/components/RequestCodePanel.astro

```diff
@@ -0,0 +1,25 @@
+---
+import { HTTPSnippet } from 'httpsnippet';
+const { title = 'Request', filename } = Astro.props;
+import { Code } from 'astro:components';
+import CodePanel from './CodePanel.astro';
+
+const file = await import(`../../code-snippets/${filename}.json`);
+const snippet = new HTTPSnippet(file);
+const request = snippet.requests[0];
+console.log({ request });
+const path = request.uriObj.pathname;
+const method = request.method;
+
+const options = { short: true, prettifyJson: true };
+const output = snippet.convert('shell', 'curl', options);
```

Comment:
@alexBinns to generate another language you just need one more line to do another conversion.
An example would be:
`const output2 = snippet.convert('node', 'request', options);`

Here is the output:

<img width="424" alt="image" src="https://github.com/centrapay/centrapay-docs/assets/64108933/2b906ae2-57a1-42e7-9976-f8c5cf01dd86">